### PR TITLE
docs: Added disabled Datepicker in a Reactive form

### PR DESCRIPTION
**Pull request by SaiShan**
#### Please provide a link to the associated issue.
#1179

### CODE REVIEW

File: docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html [hunk 1]
```diff
@@ -81,3 +81,11 @@ <h2>Disabled State</h2>
     <fd-date-picker-disabled-example></fd-date-picker-disabled-example>
 </component-example>
 <code-example [exampleFiles]="datePickerDisabled"></code-example>
+
+<separator></separator>
+<h2>Disabled Reactive Form</h2>
+<description>This example shows how to disabled Datepicker in a Reactive Form using the <code>disabled</code> attribute.</description>
```
Comment (InnaAtanasova) [inline docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html]: how to `disable` Datepicker

File: docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html [hunk 2]
```diff
@@ -81,3 +81,11 @@ <h2>Disabled State</h2>
     <fd-date-picker-disabled-example></fd-date-picker-disabled-example>
 </component-example>
 <code-example [exampleFiles]="datePickerDisabled"></code-example>
+
+<separator></separator>
+<h2>Disabled Reactive Form</h2>
```
Comment (InnaAtanasova) [inline docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html]: For consistency with the other components, could you name this `Disabled DatePicker In A Reactive Form` ?

Comment (SaiShan) [reply docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html]: Yes!

**Comment by SaiShan** [comment]
Another PR with reactive forms changes will be open.

**Pull request closed without merging**